TotalCrossVM/docker/arm32v7/Dockerfile

```diff
@@ -33,13 +32,12 @@ RUN apt-get update && \
         fcitx-libs-dev \
         libsamplerate0-dev \
         libsndio-dev \
-# Wayland
         libwayland-dev \
         libxkbcommon-dev \
         wayland-protocols \
         git \
         ca-certificates; \
-    apt-get install -y libgles1-mesa-dev || apt-get -f install; \
+    apt-get install -y libglvnd-dev || apt-get -f install; \
```

Comment:
why has this been changed?

Reply:
As I remember Ubuntu bionic was compiling about package 'libgles1-mesa-dev' and It recommends to use 'libglvnd-dev' as replacement package

Reply:
@acmlira the commit is missing this information